plugins/kafka/kafka.cpp

```diff
@@ -475,7 +475,14 @@ namespace KafkaPlugin
     {
         if (message.err() != RdKafka::ERR_NO_ERROR)
         {
-            DBGLOG("Kafka: Error publishing message: %d (%s); message: '%s'", message.err(), message.errstr().c_str(), static_cast<char*>(message.payload()));
```

Comment:
The performance isn't an issue (as you say), it's just a question of which is more maintainable. I don't have strong feelings either way, and it's your code, so I'm happy to merge as is...